esm_master/compile_info.py

```diff
@@ -97,13 +98,24 @@ def combine_components_yaml(parsed_args):
         # TODO(PG): Switch around async optional
         #get_all_package_info(os.listdir(cat_dir), cat, cat_dir, components_dict, relevant_entries, parsed_args)
     default_infos = {}
-    for i in os.listdir(DEFAULTS_DIR):
-        if parsed_args.get("verbose", 0) > 1:
-            print(f"Reading file {DEFAULTS_DIR}/{i}")
-        file_contents = esm_parser.yaml_file_to_dict(DEFAULTS_DIR + "/" + i)
+
+    # get the files in the defaults directory and exclude general.yaml
+    yaml_files = os.listdir(DEFAULTS_DIR)
+    if 'general.yaml' in yaml_files:
+        yaml_files.remove('general.yaml')
+
+    for yaml_file in yaml_files:
+        if os.getenv("ESM_MASTER_DEBUG"):
```

Comment:
Why don't we use just `verbose` here and in the other instances you changed to `ESM_MASTER_DEBUG`?

Reply:
I did that in the first commit: https://github.com/esm-tools/esm_master/pull/56/commits/3fa45e9dce3e8d259ef64107ca91260d79d91e4c

However, the output is EXTREMELY dirty. Definitely it is more like a debug level verbosity. Previously Paul told me to have an environment variable like `ESM_PARSER_DEBUG` or `ESM_MASTER_DEBUG` to separate such detailed information from the rest of the verbose outputs.

The main idea is `--verbose` should give just enough information but not anything more like debugging information. For this reason I changed the useless ones to `ESM_MASTER_DEBUG` and kept the necessary ones as `parsed_args.get("verbose", False)`

Reply:
Okay, that makes sense to me :)